syncthing_gtk/app.py

```diff
@@ -1246,13 +1246,13 @@ def hilight(self, boxes):
 					if box in f["devices"]:
 						to_hilight.add(f)
 				to_hilight.add(box)
-		for box in [] + self.devices.values() + self.folders.values():
+		for box in [] + list(self.devices.values()) + list(self.folders.values()):
```

Comment:
If we are changing this, wouldn't `itertools.chain(self.devices.values(), self.folders.values())` work in py3?

Reply:
Yes, that would work in both.  I'll do it